sdk/eventhub/event-hubs/samples/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "azure-event-hubs-samples",
+  "version": "1.0.0",
+  "description": "The samples in this folder are for version 5.0.0 and above of this library. If you are using version 2.1.0 or lower, then please use [samples for v2.1.0](https://github.com/Azure/azure-sdk-for-js/tree/%40azure/event-hubs_2.1.0/sdk/eventhub/event-hubs/samples) instead",
+  "main": "dist/runsamples.js",
+  "scripts": {
+    "samples": "ts-node ./runsamples.ts",
+    "build": "tsc -p ."
+  },
+  "keywords": [
+    "Azure",
+    "Storage",
+    "File",
+    "Node.js",
+    "TypeScript"
+  ],
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Azure/azure-sdk-for-js/issues"
+  },
+  "homepage": "https://github.com/Azure/azure-sdk-for-js#readme",
+  "devDependencies": {
+    "ts-node": "^8.5.4",
+    "typescript": "^3.7.2"
```

Comment:
We might need some version of `@types/node` as a devDependency as well.

Reply:
It's odd, I totally agree with you but I didn't notice it because no compile errors came up. Must have been coming in as a transitive dependency?

Reply:
As you pointed out it's because it's hitting the parent's node_modules folder. I moved it out (temporarily), built it and added in all the missing modules.